src/NUnitFramework/framework/Internal/Execution/SimpleWorkItem.cs

```diff
@@ -35,16 +36,20 @@ namespace NUnit.Framework.Internal.Execution
     /// </summary>
     public class SimpleWorkItem : WorkItem
     {
+        private readonly IDebugger _debugger;
+
         readonly TestMethod _testMethod;
 
         /// <summary>
         /// Construct a simple work item for a test.
         /// </summary>
         /// <param name="test">The test to be executed</param>
         /// <param name="filter">The filter used to select this test</param>
-        public SimpleWorkItem(TestMethod test, ITestFilter filter)
```

Comment:
The same things apply to `SimpleWorkItem` that applied to `TimeoutCommand`. Either this will break people and we should leave the public constructor in place, or it won't break people and therefore making the whole class internal won't break people either since I believe the class is unusable if the constructor is unusable.

For consistency we should do what we did for TimeoutCommand.